plugins/gateway/tyk/v3.2.2/plugin.go

```diff
@@ -67,7 +69,11 @@ func PostGetAPIDefinition(_ http.ResponseWriter, r *http.Request) {
 		logger.Error("Failed to get api definition")
 		return
 	}
+	// set the apiDefinition since we dont get it in the response phase
 	ctx.SetDefinition(r, apiDefinition)
+
+	// set request time on session metadata
+	ctx.SetSession(r, &user.SessionState{MetaData: map[string]interface{}{common.RequestTimeContextKey: time.Now()}}, false, false)
```

Comment:
you should not use `common.GetTimeNowRFC3339Nano()` here instead `time.Now()`?

Reply:
Done